stylize/clang_formatter.py

```diff
@@ -34,3 +35,14 @@ def run(self, args, filepath, check=False):
             proc.communicate()
             md5_after = file_md5(filepath)
             return md5_before != md5_after
+
+    def get_command(self):
+        if os.system("command -v clang-format >/dev/null") == 0:
```

Comment:
Python actually has a builtin version of `which` that you can use here:

https://docs.python.org/3/library/shutil.html#shutil.which